requirements/requirements.txt

```diff
@@ -2,7 +2,7 @@ numpy>=1.18.2
 onnx>=1.7.0
 pytorch-lightning>=1.1.0,<=1.1.5
 python-dateutil
-torch
```

Comment:
we can't pin torch to the exact version because it will force torch re-install if the user has slightly different version (e.g. 1.7.0) or inside NVIDIA's container (20.11-py3) which has version like (1.8.0a0+17f8c32 --- this is before upstream's 1.8)

Reply:
@okuchaiev thanks for the suggestion. 
I tried to let torch as it was and just changed the range for `pytorch-lightning>=1.2.0,<=1.2.2`.
At this moment 1.2.2 is the latest and 1.2.0 seems to have some PyTorch 1.8 compatibility fixes.

I recreate the env from the scratch with `./reinstall.sh` with the above PLT range changes, but the issue is still there. Cannot choose newer version from PLT.
Then I found somewhere a workaround like
```
torchtext>=0.5, <0.7 # TODO: temporary fix fix for compatibility
```
I downgraded `torchtext` to 0.7, then complained about `_torchtext.so` undefined symbol. Probably it will not be possible to downgrade just `torchtest`.

I have no other idea, then just for myself I come back to torch 1.7.1 and if somebody will install these days, may run in the same issue and do like myself.

Reply:
Now, `pytorch-lightening` 1.2.3 came out, which has the fix. I did a test. This PLT version incompatibility is fixed, but came out another. You can read https://github.com/NVIDIA/NeMo/issues/1868